File: backend/backend/Model/Quiz.cs
using backend.Constants;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace backend.Model
{
    [Table("Quiz")]
    public class Quiz
    {

        [Key]
        public Guid Id { get; set; }
        public Category Category { get; set; }
        public int Year { get; set; }
        public string Subject { get; set; }
        public IEnumerable<Question>? Question { get; set; }

    }
}

File: scripts/parseSubjectPdf/Model/Quiz.py
from Model.Question import Question


class Quiz:

    def __init__(self, subject: str, year: int, question=list()) -> None:
        self._subject = subject
        self._year = year
        self._question = question

    def get_subject(self):
        return self._subject

    def get_year(self):
        return self._year

    def get_question(self):
        return self._question

    def add_question(self, question: Question):
        self._question.append(question)
